api/guardrails.yaml

```diff
@@ -0,0 +1,703 @@
+# Copyright (c) Ultraviolet
+# SPDX-License-Identifier: Apache-2.0
+
+openapi: 3.0.0
+info:
+  title: Cube Guardrails API
+  description: API for managing guardrail configurations and processing chat completions through NeMo Guardrails
+  version: 1.0.0
+  contact:
+    name: Ultraviolet
+    url: https://github.com/ultravioletrs/cube
+
+servers:
+  - url: http://localhost:8001
+    description: Local development server
+  - url: https://api.example.com
+    description: Production server
+
+paths:
+  /v1/chat/completions:
```

Comment:
this path is proxied we don't need to define it here since users will never reach this endpoint directly, only entry point to cube is proxy so update the other paths as well